pyteal/ast/int.py

```diff
@@ -30,7 +30,7 @@ def __init__(self, value: int) -> None:
 
     def __teal__(self, options: "CompileOptions"):
         op = TealOp(self, Op.int, self.value)
-        return TealBlock.FromOp(options, op)
+        return TealBlock.FromOp(options, op, traceback=self.trace)
```

Comment:
Forced to pass traceback into blocks or ops, so this would need to happen for every Expr